changelog/unreleased/pull-2733

```diff
@@ -0,0 +1,7 @@
+Enhancement: Enable support for EKS IRSA
+
+This change enables restic to use S3 as a backend using the EKS IAM Roles For
```

Comment:
The official capitalization seems to be "IAM roles for service accounts". I'd reword the sentence to "We've added support for EKS IAM roles for service accounts feature to the S3 backend."